swarm/storage/mru/resource_test.go

```diff
@@ -681,7 +687,7 @@ func TestValidator(t *testing.T) {
 		t.Fatalf("sign fail: %v", err)
 	}
 
-	chunk, err = newUpdateChunk(&mr.SignedResourceUpdate)
+	chunk, err = mr.SignedResourceUpdate.newUpdateChunk()
```

Comment:
It seems strange to me to have this as a method of `SignedResourceUpdate`. The description of which being:

```
// SignedResourceUpdate contains signature information about a resource update
```

Reply:
I am changing the description of `SignedResourceUpdate` to "SignedResourceUpdate represents a resource update with all the necessary information to prove ownership of the resource"